flow/layers/layer.h

```diff
@@ -89,6 +90,7 @@ class Layer {
     TextureRegistry& texture_registry;
     const RasterCache* raster_cache;
     const bool checkerboard_offscreen_layers;
+    bool has_platform_view;
```

Comment:
Not sure why this new field is necessary. If the raster cache needs to be disabled, the raster_cache pointer can just not be supplied in the paint context. There is nothing more to do.

Reply:
It's not necessary, removed.